### PR TITLE
Few missing language types in .cs

### PR DESCRIPTION
**Pull request by jtkech**

### CODE REVIEW

**Comment by MikeAlhayek** [comment]
Oops, these must have been merges after the branch was created! Good catch

**Comment by jtkech** [comment]
No problem

File: src/OrchardCore.Modules/OrchardCore.Workflows/Http/Services/WorkflowInstanceRouteEntries.cs [hunk 1]
```diff
@@ -63,7 +63,7 @@ internal static IEnumerable<WorkflowRoutesEntry> GetWorkflowRoutesEntries(Workfl
                 var activity = activityLibrary.InstantiateActivity<HttpRequestFilterEvent>(x);
                 var entry = new WorkflowRoutesEntry
                 {
-                    WorkflowId = workflow.WorkflowId,
+                    WorkflowId = workflow.Id.ToString(),
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Workflows/Http/Services/WorkflowInstanceRouteEntries.cs:66]: Why this change?

**Review by MikeAlhayek — APPROVED** [review]

File: src/OrchardCore.Modules/OrchardCore.Workflows/Http/Services/WorkflowInstanceRouteEntries.cs (hunk 1 above)
Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.Workflows/Http/Services/WorkflowInstanceRouteEntries.cs:66]: This is an old mistake, it couldn't work as somewhere else it is doing a `long.Parse()` on it. But `WorkflowId` is not a long, this is `Id` which is a long.

Here in `GetWorkflowRoutesEntries()` I did the same as in `GetWorkflowTypeRoutesEntries()`.

Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.Workflows/Http/Services/WorkflowInstanceRouteEntries.cs:66]: Not sure I follow you. You know better. But maybe fix that is a separate PR

Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.Workflows/Http/Services/WorkflowInstanceRouteEntries.cs:66]: Okay will remove asap

Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.Workflows/Http/Services/WorkflowInstanceRouteEntries.cs]: Done
